freetds.opam

```diff
@@ -0,0 +1,38 @@
+opam-version: "1.2"
+
+maintainer: "Christophe.Troestler@umons.ac.be"
+homepage: "https://github.com/kennknowles/ocaml-freetds"
+dev-repo: "https://github.com/kennknowles/ocaml-freetds.git"
+bug-reports: "https://github.com/kennknowles/ocaml-freetds/issues"
+license: "LGPL-2.1"
+
+tags: [
+  "clib:ct"
+  "clib:sybdb"
+]
+build: [
+  [ "./autogen.sh" ]
+  [ "./configure"
+    "--prefix" prefix
+    "--sbindir=%{lib}%/freetds/sbin"
+    "--libexecdir=%{lib}%/freetds/libexec"
+    "--sysconfdir=%{lib}%/freetds/etc"
+    "--sharedstatedir=%{lib}%/freetds/com"
+    "--localstatedir=%{lib}%/freetds/var"
+    "--libdir=%{lib}%/freetds/lib"
+    "--includedir=%{lib}%/freetds/include"
+    "--datarootdir=%{lib}%/freetds/share" ]
+  [ "make" ]
+]
+install: [ "make" "install" ]
+remove: [[ "ocamlfind" "remove" "freetds" ]]
+depends: [
+  "ocamlfind" {build}
+]
+depexts: [
+  [["centos"] ["freetds-devel"]]
+  [["debian"] ["freetds-dev"]]
+  [["fedora"] ["freetds-devel"]]
+  [["ubuntu"] ["freetds-dev"]]
+  [["osx" "homebrew"] ["freetds"]]
+]
```

Comment:
There is a `\n` at the end of this line, but GitHub isn't showing it.

Reply:
It does now (it was complaining there was none before).